app/src/main/java/com/yeonae/chamelezone/view/place/presenter/PlaceReviewPresenter.kt

```diff
@@ -14,7 +15,11 @@ class PlaceReviewPresenter(private val reviewRepository: ReviewRepository,
     override fun placeDetailReview(placeNumber: Int) {
         reviewRepository.getReviewList(placeNumber, object : ReviewCallBack<List<ReviewResponse>> {
             override fun onSuccess(response: List<ReviewResponse>) {
-                placeReviewView.showPlaceReview(response)
+                val reviewItem = arrayListOf<ReviewItem>()
```

Comment:
```suggestion
                val reviewItemList = arrayListOf<ReviewItem>()
```
단일 객체를 의미하는 네이밍에 변경이 필요한 것 같습니다.

Reply:
수정했습니다
d19bab2